frontend/src/pages/RegisterUser.tsx

```diff
@@ -56,66 +56,62 @@ export default class RegisterCustomer extends React.Component<
 
 	render() {
 		return (
-			<Form>
+			<div className="registerBody">
```

Comment:
@vong98 Lets make a generic CSS class like "content-body" and if there are register page-specific CSS isolate that CSS into the smallest CSS class possible and chain them like this https://stackoverflow.com/questions/36772389/how-can-i-add-multiple-classnames-to-react-component?rq=1